lib/file.js

```diff
@@ -634,6 +637,15 @@ export type EntitySet<T> = T[] & {
 export type DeepRequired<T> = { 
     [K in keyof T]: DeepRequired<T[K]>
 } & Exclude<Required<T>, null>;
+
+
+/**
+ * Dates and timestamps are strings during runtime, so cds-typer represents them as such.
+ */
+export type CdsDate = ${dateRegex};
+export type CdsDateTime = string;
+export type CdsTime = ${timeRegex};
+export type CdsTimestamp = string;
```

Comment:
```suggestion
export type CdsTimeStamp = string;
```

Reply:
ah, actually this should be the other way around to be consistent with the [built-in types](https://pages.github.tools.sap/cap/docs/guides/domain-modeling#standard-built-in-types). Good that you noticed the discrepancy there.